### PR TITLE
Display how many customer has been anonymized and fix dateTime object anonymization

### PR DESCRIPTION
**Pull request by GalloisLuca**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?   | no

### CODE REVIEW

File: src/Processor/AdvancedActions/AnonymizeCustomersNotLoggedBeforeProcessor.php [hunk 1]
```diff
@@ -22,11 +23,15 @@ class AnonymizeCustomersNotLoggedBeforeProcessor implements AdvancedActionsFormD
     /** @var ParameterBagInterface */
     private $parameterBag;
 
-    public function __construct(EntityManagerInterface $entityManager, AnonymizerProcessor $anonymizerProcessor, ParameterBagInterface $parameterBag)
+    /** @var FlashBagInterface */
+    private $flashBag;
```
Comment (TheGrimmChester) [inline src/Processor/AdvancedActions/AnonymizeCustomersNotLoggedBeforeProcessor.php]: You can type hint that

File: src/Processor/AdvancedActions/AnonymizeCustomersWithoutAnyOrdersBeforeProcessor.php [hunk 2]
```diff
@@ -23,11 +24,15 @@ class AnonymizeCustomersWithoutAnyOrdersBeforeProcessor implements AdvancedActio
     /** @var ParameterBagInterface */
     private $parameterBag;
 
-    public function __construct(EntityManagerInterface $entityManager, AnonymizerProcessor $anonymizerProcessor, ParameterBagInterface $parameterBag)
+    /** @var FlashBagInterface */
+    private $flashBag;
```
Comment (TheGrimmChester) [inline src/Processor/AdvancedActions/AnonymizeCustomersWithoutAnyOrdersBeforeProcessor.php]: You can type hint that

File: src/Processor/AnonymizerProcessor.php [hunk 3]
```diff
@@ -17,12 +19,25 @@ final class AnonymizerProcessor
     /** @var EntityManagerInterface */
     private $entityManager;
 
+    /** @var TranslatorInterface */
+    private $translator;
```
Comment (TheGrimmChester) [inline src/Processor/AnonymizerProcessor.php]: You can type hint that

File: src/Processor/AnonymizerProcessor.php [hunk 4]
```diff
@@ -17,12 +19,25 @@ final class AnonymizerProcessor
...
+
+    /** @var LoggerInterface */
+    private $logger;
```
Comment (TheGrimmChester) [inline src/Processor/AnonymizerProcessor.php]: You can type hint that

File: src/Processor/AnonymizerProcessor.php [hunk 5]
```diff
@@ -17,12 +19,25 @@ final class AnonymizerProcessor
...
+
+    /** @var int */
+    private $anonymizedEntity = 0;
```
Comment (TheGrimmChester) [inline src/Processor/AnonymizerProcessor.php]: You can type hint that

**Review by TheGrimmChester — CHANGES_REQUESTED** [review]

File: src/Processor/AdvancedActions/AnonymizeCustomersNotLoggedBeforeProcessor.php (hunk 1 above)
Comment (GalloisLuca) [reply src/Processor/AdvancedActions/AnonymizeCustomersNotLoggedBeforeProcessor.php:27]: No i can't, else the compatibility with php7.3 won't work anymore

**Comment by oallain** [comment]
Why doesn't this PR target **master**?

**Review by oallain — DISMISSED** [review]

**Review by TheGrimmChester — APPROVED** [review]

**Comment by oallain** [comment]
Thanks @GalloisLuca and @tuyetrinhvo 👍🏼

**Pull request merged**